app-demo/src/main/res/layout/activity_demo_styled_switch.xml

```diff
@@ -30,6 +33,12 @@
     <co.sodalabs.view.StyledSwitch
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        app:on="false"
```

Comment:
Could make the label more specific by adding the preview, such as `sw`. As we have more custom attributes, the management of the naming conflict becomes important. To minimize the naming conflict, we could simply add the prefix to the custom attributes. I know the attributes are already scoped under `StyledSwitch`, and it is scoped programmatically. Unfortunately, in XML, it is likely to conflict unless we specify the namespace.

Reply:
In addition to that, I quite like the `textOn` and `textOff` serving specifically for the label.